twitchpopups.htm

```diff
@@ -145,6 +197,9 @@ <h1 id="popuptext"></h1>
     // Animate off
     const deleteAnimation = () => {
         hotSeatIsOn = false;
+        votingIsOn = false;
+        options = [];
+        votes = {};

```

Comment:
I don't think you need this

Reply:
This is needed, at the minute the deleteAnimation is the general close and clean up command. The alternative would be to have a separate deleteAnimationHotseat() and deleteAnimationVotes() but I think that would be needlessly complex, resolve the conversation if you agree!

Reply:
GitHub didn't make it very clear, but I was commenting specifically on line 202. votes isn't referenced anywhere else in the code. not that I can see anyway.

Reply:
sorted this, thanks for reopening!